examples/seismic/tti/operators.py

```diff
@@ -512,5 +526,116 @@ def AdjointOperator(model, geometry, space_order=4,
     return Operator(stencils, subs=model.spacing_map, name='AdjointTTI', **kwargs)
 
 
+def JacobianOperator(model, geometry, space_order=4,
+                     **kwargs):
+    """
+    Construct a Linearized Born operator in a TTI media.
```

Comment:
This is missing a blank line before "Parameters"

Reply:
the line was added.